code/modules/projectiles/projectile/bullets/shotgun.dm

```diff
@@ -1,7 +1,8 @@
 /obj/projectile/bullet/shotgun_slug
 	name = "12g shotgun slug"
-	damage = 50
+	damage = 35
 	sharpness = SHARP_POINTY
+	armour_penetration = 80
```

Comment:
Weren't you going to change this?

Reply:
oh crap, for some reason it didnt change to 30